src/borrow_tracker/stacked_borrows/mod.rs

```diff
@@ -693,12 +494,16 @@ impl Stacks {
     ) -> InterpResult<'tcx> {
         trace!("deallocation with tag {:?}: {:?}, size {}", tag, alloc_id, range.size.bytes());
         let dcx = DiagnosticCxBuilder::dealloc(machine, tag);
-        let state = machine.stacked_borrows.as_ref().unwrap().borrow();
+        let state = machine.borrow_tracker.as_ref().unwrap().borrow();
         self.for_each(range, dcx, |stack, dcx, exposed_tags| {
             stack.dealloc(tag, &state, dcx, exposed_tags)
         })?;
         Ok(())
     }
+
+    pub fn expose_tag(&mut self, tag: BorTag) {
```

Comment:
```suggestion
    fn expose_tag(&mut self, tag: BorTag) {
```
This is only called in this file, right?